pyproject.toml

```diff
@@ -1,65 +1,57 @@
 [build-system]
-requires = [
-    "setuptools>=61.0.0",
-]
+requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "nomad-analysis"
-version = '0.1.0'
```

Comment:
I would setup dynamic versioning here like we have in `nomad-material-processing` and `nomad-measurements` packages.

Reply:
Yeah, agree, I will change it to `dynamic = ["version"]`.